src/components/StripeBankAccountTokenInputField/StripeBankAccountRequiredInput.js

```diff
@@ -43,7 +44,7 @@ const StripeBankAccountRequiredInput = props => {
   const errorMessage = <p className={css.error}>{inputError}</p>;
 
   return (
-    <div className={classes}>
+    <div className={wrapperClassName ? wrapperClassName : classes}>
```

Comment:
I think the existing pattern of `rootClassName` is the correct way to deal with this.

If success/error classes should not be used, then those classes could be used through prop that disables them (e.g. `useValidationClasses=<true/false>`)